bin/install_npm_packages

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -eu
+
+# This script expects a valid package.json at the project's root
+if [ ! -f package.json ]; then
```

Comment:
We should instead check for the presence of `package-lock.json` – if that's not present, it's not really safe to run `npm install` (because this could be a yarn repo with a `yarn.lock` file).

It'd be fine to check for `package.json` as well, but the `lock` file is the more important thing.

Reply:
Oh yes, that makes a lot of sense! I've added the check for the package-lock.json file in b95b0639794068b7715dea0474893da0bbceea60

Reply:
I'm not sure this is correct – these should be two separate operations:

1. Ensure package.json is present (ie – is this a node project?)
2. Ensure package-lock.json is present (ie – is this an npm project (as opposed to a yarn project))

Reply:
Assuming you meant that we should check for **both** conditions, not just one or the other. e6e45de15a28ccd21f2fd8ef25e9ef60ab9b5697 now splits the check into two separate if-statements to ensure **both** files are present.